examples/ping-json/README.md

```diff
@@ -22,7 +22,7 @@ Lookup the node `my_key` key belongs to using [tcurl][2]:
 Call the `/ping` endpoint (multiple times) and see the request being forwarded. Each request is sent to a random node in the cluster because of the `-P hosts.json` argument--but is always handled by the node owning the key. This can be seen in the `from` field of the response:
 
     $ tcurl pingchannel -P hosts.json /ping '{"key": "my_key"}'
```

Comment:
The pheader is already in the second test below this one. This example/test is without the header, but it was failing because the repsonse always contains the `pheader` even when it is empty.